server/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+	"github.com/coreos/etcd/clientv3"
+	"github.com/jessevdk/go-flags"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const (
+	// Subdirectories of our data dir
+	etcdDir      = "etcd"
+	containerDir = "container"
+)
+
+var opts struct {
+	Args struct {
+		Ip   string `description:"Public IP to bind"`
```

Comment:
`IP` - no CamelCasing for acronyms (https://github.com/golang/go/wiki/CodeReviewComments#initialisms)